TrackerConditions/src/StrawResponse.cc

```diff
@@ -392,4 +387,31 @@ namespace mu2e {
 
     }
 
+   double StrawResponse::BTrk_driftDistanceToTime(StrawId strawId, double ddist, double phi) const {
+     return _strawDrift->D2T(ddist,phi);
+  }
+
+   double StrawResponse::BTrk_driftInstantSpeed(StrawId strawId, double ddist, double phi) const {
+     return _strawDrift->GetInstantSpeedFromD(ddist);
+   }
+
+   double StrawResponse::BTrk_driftDistanceError(StrawId strawId, double ddist, double phi) const {
+     static const std::vector<double> derr = {0.363559, 0.362685, 0.359959, 0.349385,
```

Comment:
Just a comment for future referende.  This could be std::array.  No need to actually change it here since it will go away soon.